src/wp-admin/site-editor.php

```diff
@@ -88,7 +88,8 @@ static function ( $classes ) {
 );
 
 $preload_paths = array(
-	array( '/wp/v2/media', 'OPTIONS' ),
+	array( rest_get_route_for_post_type_items( 'attachment' ), 'OPTIONS' ),
```

Comment:
Avoids hardcoding paths. Matches the post editor.

https://github.com/WordPress/wordpress-develop/blob/b20e852a56c0970684a44166fcffa3d041f05eb8/src/wp-admin/edit-form-blocks.php#L61